client/network_connect_test.go

```diff
@@ -87,6 +87,10 @@ func TestNetworkConnect(t *testing.T) {
 				return nil, fmt.Errorf("expected 'container_id', got %s", connect.Container)
 			}
 
+			if connect.EndpointConfig == nil {
```

Comment:
Hum, the test will fail both way (either an error, or a panic I guess). I guess it's ok to add this check, the failure would be more explicit 👼